Assets/Scripts/TodoList.cs

```diff
@@ -67,47 +74,35 @@ public void FillList()
         }
     }
 
-    public void PrintList()
+    /// <summary>
+    /// Fills all the images in the UI
+    /// </summary>
+    public void LoadList()
     {
-        ////if (gameManager.listActive())
-        ////{//if we want the list to show
-        //    text.text = "List: \n";
-        //    foreach (KeyValuePair<PickUpAbles, bool> kvp in list)
-        //    {//foreach item in list
-        //        text.text += '\n' + kvp.Key.gameObject.name; //add name to list
-        //    }
-        ////} else
-        //{//if we dont
-        //  //  text.text = "";
-        //}
-
-        Debug.Log(list.Count);
-
-        FillImage(image);
-        Debug.Log("2");
-        FillImage(image2);
-        Debug.Log("3");
-        FillImage(image3);
-        //foreach(Image image in images)
-        //{
-        //    FillImage(image);
-        //}
+        foreach(Image image in images)
+        {
+            FillImage(image);
+        }
     }
 
+
     void FillImage(Image img)
-    {
+    {//TODO make it so the images don't get repeated
```

Comment:
remove todo if completed, otherwise fix it

Reply:
Completed